README.md

```diff
@@ -1,9 +1,10 @@
 [![CircleCI](https://circleci.com/gh/klaytn/klaytn/tree/master.svg?style=svg&circle-token=f7008cc62f0f16070af2ac3bd8f34989a9fd5724)](https://circleci.com/gh/klaytn/klaytn/tree/master)
 [![codecov](https://codecov.io/gh/klaytn/klaytn/branch/master/graph/badge.svg?token=Tb7cRhQUsU)](https://codecov.io/gh/klaytn/klaytn)
+[![](https://godoc.org/github.com/klaytn/klaytn?status.svg)](http://godoc.org/github.com/klaytn/klaytn)
```

Comment:
How about this?
`[![GoDoc](https://godoc.org/github.com/klaytn/klaytn?status.svg)](http://godoc.org/github.com/klaytn/klaytn)`